api/src/main/java/com/velocitypowered/api/event/proxy/ListenerCloseEvent.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2018 Velocity Contributors
+ *
+ * The Velocity API is licensed under the terms of the MIT License. For more details,
+ * reference the LICENSE file in the api top-level directory.
+ */
+
+package com.velocitypowered.api.event.proxy;
+
+import com.google.common.base.Preconditions;
+import java.net.InetSocketAddress;
+
+/**
+ * This event is fired by the proxy before the proxy stops accepting connections.
+ */
+public final class ListenerCloseEvent {
+
+  private final InetSocketAddress address;
```

Comment:
Include the type here too?

Reply:
The close() method in ConnectionManager does not include type, so I'm not quite sure what to do with that

Reply:
Then add it.

Reply:
Oh, my bad - I thought it was a public api.